app/controllers/application_controller.rb

```diff
@@ -41,6 +41,12 @@ def handle_unverified_request
 
   def set_affiliate
     @affiliate = Affiliate.active.find_by_name(permitted_params[:affiliate])
```

Comment:
What do you think about removing the `active` scope in here, just find the affiliate. And in here https://github.com/GSA/search-gov/pull/1845/files#diff-b8f829c906439ddf15f11f1d3c26a93cf538f739dff9e107d46df8768de16aacR19 to check if the affiliate is active or not instead?
Removing the need of a second query and more other changes.